ui/src/components/textInputDialog.tsx

```diff
@@ -20,7 +21,9 @@ export function useTextInputDialog(
   const [open, setOpen] = useState(false);
   const show = () => setOpen(true);
 
-  const [text, setText] = useState(props.title);
+  const [text, setText] = useState(
+    typeof props.text !== "undefined" ? props.text : props.title
```

Comment:
Putting the title in the text box doesn't make sense since the title would be a weird thing to use as a name (e.g. "Edit Criteria Name", though I know that wouldn't happen in this case). Instead use `props.text || ""`.

Reply:
What more specifically was the problem? I would have thought that `props.text || ""` would have the same behavior as currently: empty for new cohorts and existing text for renaming criteria. Alternatively, if the integration test is expecting the text box to contain the title then the test should be fixed.

Reply:
Yes... my fault